plutus-tx-plugin/src/PlutusTx/Options.hs

```diff
@@ -169,6 +170,7 @@ pluginOptions =
                     , Implication (== False) posRelaxedFloatin True
                     , Implication (== False) posPreserveLogging False
                     , Implication (== False) posCaseOfCaseConservative False
+                    , Implication (== False) posInlineConstants False
```

Comment:
This is "conservative-optimization == False implies inline-constants == False", but it should be the other way around. Also there should be another entry for `Implication (== True)`.